process.py

```diff
@@ -451,7 +458,7 @@ def process_arguments()->Tuple[TextIO, TextIO, bool, bool, bool]:
     print_tree = args.verbose
     only_one_parent = args.unique
     missing_feature_ok = args.missing
-    return in_file, out_file, dot_file, print_tree, only_one_parent, missing_feature_ok
+    return in_file, out_file, dot_file, print_tree, only_one_parent, missing_feature_ok,file_reader
```

Comment:
Please be mindful of formatting. Need a space after the last comma.

Reply:
Fixed formatting